change/@fluentui-web-components-2021-01-04-10-08-52-users-v-sedono-add-menu-disabled-example.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

Comment:
@SethDonohue I'm not actually sure this needs to be a patch. This can likely be the equivalent of a chore as it does not affect distributed packages. Can you update the change file here?

Reply:
Yes, and I'll check these on all of the recent PRs, I over looked this.

Reply:
Thx - I think this and the storybook one may be the only ones I saw with the issue

Reply:
updated